src/SFA.DAS.AssessorService.Web/Validators/Withdrawal/CheckWithdrawalRequestViewModelValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentValidation;
+using SFA.DAS.AssessorService.Web.ViewModels.ApplyForWithdrawal;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class CheckWithdrawalRequestViewModelValidator : AbstractValidator<CheckWithdrawalRequestViewModel>
+    {
+        public CheckWithdrawalRequestViewModelValidator()
+        {
+            RuleFor(vm => vm.Continue)
+                .Must(value => !string.IsNullOrEmpty(value) && (value.ToLower() == "yes" || value.ToLower() == "no"))
```

Comment:
I think this is slightly more readable, while also accounting for all-whitespace strings.
```suggestion
            RuleFor(vm => vm.Continue)
                .Cascade(CascadeMode.Stop) // don't evaluate the Must if null or empty
                .NotEmpty() // 
                .Must(value => value.ToLower() == "yes" || value.ToLower() == "no");
```

Reply:
Good suggestion accepted that - you had removed the message is your version though